DisplayBundle/EventSubscriber/SendMessageSubscriber.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace OpenOrchestra\DisplayBundle\EventSubscriber;
+
+use OpenOrchestra\DisplayBundle\Event\MailerEvent;
+use OpenOrchestra\DisplayBundle\MailerEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Class SendMessageSubscriber
+ */
+class SendMessageSubscriber implements EventSubscriberInterface
+{
+    protected $messages;
+    protected $mailer;
+
+    /**
+     * @param \Swift_Mailer $mailer
+     */
+    public function __construct(\Swift_Mailer $mailer)
+    {
+        $this->messages = array();
```

Comment:
this declaration can be directly done when you declare the property : 
protected $messages = array();